src/server/index.jsx

```diff
@@ -165,41 +162,38 @@ server
       });
     },
   )
-  .get(
-    [articleRegexPath, frontpageRegexPath, ...mediaRadioAndTvRegexPathsArray],
-    async ({ url, headers, path: urlPath }, res) => {
+  .get('/*', async ({ url, headers, path: urlPath }, res) => {
```

Comment:
`path` is renamed to `urlPath` here and then renamed to `path` again [here](https://github.com/bbc/simorgh/pull/2902/files#diff-788dcef7a5e3a2740bf71979cca64a59R180)

Reply:
:see I havent touched this code, the diff is just weird, so dont want to change it now due to the size of the PR

Reply:
ah ok cool. Didn't realise it was just whitespace changes. Viewing it with a `?w=1` at the end of the the url like `https://github.com/bbc/simorgh/pull/2902/files?w=1` removes the white space diffs for easier reviewing of only your changes 👍